src/main.ts

```diff
@@ -29,4 +30,16 @@ const router = createRouter({
   routes,
 });
 
-createApp(App).use(router).mount('#app');
+const app = createApp(App);
+app.config.warnHandler = (msg, vm, trace) => {
+  // We are using logical components, so ignoring this warning
+  if (msg.includes('Component is missing template or render function')) {
```

Comment:
Is there a way to configure Vue to suppress this warning? Relying on the message is liable to having it come back when we upgrade Vue if someone decides to change the wording.

Reply:
Yeah I don't like this solution either, but I googled and honestly, this is the only way I could find. If you find a way, then feel free to fix it. Hopefully, in future Vue versions, there will be a way

Reply:
Can we add a dummy render function? In React, you can return null for it to do nothing. I would prefer to not globally disable warnings for a local usage.